src/main/java/tech/units/tck/tests/unit/UnitInterfaceTest.java

```diff
@@ -184,13 +184,25 @@ public void testUnit42121A3_DivideByDouble() {
             testHasPublicMethod("Section 4.2.1.2.1", type, Unit.class, "divide", double.class);
         }
     }
+
+	/**
+	 * Ensure the divide() operation is implemented.
+	 */
+	@SpecAssertion(section = "4.2.1.2", id = "42121-A4")
+	@Test(groups = { CORE }, description = "4.2.1.2.1 Ensure the divide(Number) operation is implemented.")
+	public void testUnit42121A4_DivideByNumber() {
+		for (@SuppressWarnings("rawtypes")
+			Class type : TCKSetup.getConfiguration().getUnitClasses()) {
+			testHasPublicMethod("Section 4.2.1.2.1", type, Unit.class, "divide", Number.class);
+		}
+	}
     
     /**
      * Ensure the multiply() operation is implemented.
      */
-    @SpecAssertion(section = "4.2.1.2", id = "42121-A4")
+    @SpecAssertion(section = "4.2.1.2", id = "42121-A5")
```

Comment:
I know, changing the numbering of test case scenarios is not optimal, but I believe it is better to keep the related ones together. In addition, the added methods are not something that will happen every day, so, for once, it can happen like that, I guess.